packages/command/test/unit/sfdxFlags.test.ts

```diff
@@ -49,6 +49,18 @@ describe('SfdxFlags', () => {
       expect(rv.apiversion).to.have.property('description', messages.getMessage('apiversionFlagDescription'));
     });
 
+    it('should carry forward additional properties on builtins when forced (for legacy toolbelt compatibility)', () => {
+      const rv = buildSfdxFlags(
+        {
+          // @ts-ignore force setting the char to simulate a legacy toolbelt use case
+          apiversion: flags.builtin({ char: 'a' })
```

Comment:
This is an example of how I plan to use this part of the PR in toolbelt to force-override a char value on a builtin flag type.  This is not something we want to encourage in the type system, rather to force the compiler to accept for the one toolbelt snowflake that requires it.